pkg/mpc/cmd/root.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"log"
+	"os"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Version: "0.0.1",
```

Comment:
This may work for a tag but for `sha`s is not working, I saw this already with the client where it gets unknown and I added this to the cli and it's also coming as nil

Reply:
it probably gets unknown because you built it with `go build ./pkg/mpc` (and didn't pass -ldflags)

Reply:
I was building with `make` and like I said, the same happens with the agent string for the client, it gets set to unkown